### DISCUSSION TITLE
ZOWE CLI host not recognised

### DISCUSSION THREAD

**Discussion by JohnBMCR** [zowe/zowe-cli#1518] [Q&A] [answered]
I'm trying to run the following cmd from ZOWE CLI in VSCODE/Terminal 

            zowe zosmf check status --host "https://zosmf.ouraddress"

... and I get ....
      
            Command Error:
            z/OSMF REST API Error:
            http(s) request error event called
            Error: getaddrinfo ENOTFOUND https://zosmf.ouraddress
            Error Details:
            HTTP(S) client encountered an error. Request could not be initiated to host.
            Review connection details (host, port) and ensure correctness.
            
            Protocol:  https
            Host:      https://zosmf.ouraddress
            Port:      443
            Base Path:
            Resource:  /zosmf/info
            Request:   GET
            Headers:   [{"X-CSRF-ZOSMF-HEADER":true}]
            Payload:   undefined

I have also included --user and --password options -but with no joy ! 

**However**, when I issue the corresponding CURL cmd (albeit from Windows) the details are returned successfully. 

            curl -k -H "Accept: application/json" -H "X-CSRF-ZOSMF-HEADER: true"  "https://zosmf.ouraddress"

**Comment by zFernand0** [comment, marked as answer]
I believe the protocol should be omitted in the `--host` property 😋

**Reply by JohnBMCR** [reply]
Thank You, yes appears that 

`zowe zosmf check status `

on its own works.
